MathLib/ODE/OdeSolverFactory.h

```diff
@@ -162,24 +161,24 @@ class ConcreteOdeSolver final
 		Implementation::setTolerance(abstol, reltol);
 	}
 
-	void setIC(const double t0, const Arr& y0) override
+	void setIC(const double t0,
+	           const MappedConstVector<NumEquations>& y0) override
```

Comment:
Only curious: Why do you replace `Arr` with `MappedConstVector` and not with some `Eigen::Matrix<double,N,1>`?